crypto/dh/dh_check.c

```
@@ -152,6 +152,10 @@ int DH_check(const DH *dh, int *ret)
     if (nid != NID_undef)
         return 1;
 
+    /* Don't do any checks at all with an excessively large modulus */
+    if (BN_num_bits(dh->params.p) > OPENSSL_DH_CHECK_MAX_MODULUS_BITS)
+        return 0;
```

IMO we should raise an error here.

---

Ok, done.